### ISSUE TITLE
pyfda-0.0.14 won't start

### ISSUE THREAD

**Issue by vuki** [chipmuenk/pyFDA#133]
This is what I get after updating to 0.1.4 and trying to start pyfdax:

```
[Errno 2] No such file or directory: 'c:\\winprog\\python36\\lib\\site-packages\\pyfda\\pyfda.conf'
[   INFO] [pyfda.pyfda_rc:192] Using 'DejaVu Sans' font.
[CRITICAL] [pyfda.filter_tree_builder:371] Config file "C:\Users\Greg\.pyfda\pyfda.conf" cannot be read.
```

**Comment by chipmuenk** [comment]
Oops, should have read your mail a bit more thoroughly ... I'll look into that.

Does the file 'c:\\winprog\\python36\\lib\\site-packages\\pyfda\\pyfda.conf' exist?

Does the file / directory "C:\Users\Greg\.pyfda\pyfda.conf" exist? Is the directory writable?

**Comment by chipmuenk** [comment]
I forgot to add the file
 https://github.com/chipmuenk/pyFDA/blob/develop/pyfda/pyfda.conf
to the PyPI setup, that's why it's missing in your installation.
As a workaround, you can copy this file from the GitHub repo to your user config directory. If this file exists, pyfda doesn't look for the file in the installation dir.

I'll upload a new release asap - thanks for reporting!

**Comment by vuki** [comment]
Yes, that was it, pyfda.conf missing from the distribution. I didn't have any pyfda.conf in my user directory.

I added the file and now pyfdax starts. Thanks.
